custom_components/spook/services.yaml

```diff
@@ -224,6 +224,25 @@ homeassistant_enable_device:
       selector:
         device:
 
+homeassistant_update_entity_id:
+  name: Update an entity's ID 👻
+  description: >-
+    Updates an entity's ID on the fly.
+  fields:
+    entity_id:
+      name: Entity
+      description: The entity/entities to update.
+      required: true
+      selector:
+        entity:
+          multiple: false
```

Comment:
The service doesn't seem to support multiple?

```suggestion
```

Reply:
Done